EIPS/eip-1.md

```diff
@@ -232,6 +232,64 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
 ^https://github.com/ethereum/devp2p/blob/[0-9a-f]{40}/.*$
 ```
 
+### Other Well Known Standards Bodies
+
+Links to other well known standards bodies specification may be included using normal markdown syntax. Be mindful of the requirements set for each SDO as these were chosen in order to maintain links to stable URLs.
+
+#### World Wide Web Consortium (W3C)
+
+Links to a W3C "Recommendation" status specification may be included using normal markdown syntax, such as:
```

Comment:
```suggestion
Links to a W3C "Recommendation" status specification may be included using normal markdown syntax, where the content of the link is the title of the specification. For example, the following link would be allowed:
```
We should require a consistent linking style for, well, consistency.

Reply:
Agreed with this consistency - I'll update the example as well to follow along with this additional style rule